public/app/features/dashboard-scene/settings/VersionsEditView.tsx

```diff
@@ -92,6 +94,26 @@ export class VersionsEditView extends SceneObjectBase<VersionsEditViewState> imp
     return sceneGraph.getTimeRange(this._dashboard);
   }
 
+  public async onRestore(version: DecoratedRevisionModel) {
+    //todo: how do we actually do the restore? do we restore as soon as we press modal btn?
+    // or do we need to save dashboard and exit edit mode?
+    await historySrv.restoreDashboard(version.uid, version.version);
```

Comment:
This request will update the json model in the db so restoration happens on the version settings, pretty much like it happens in the old arch. Do we want to keep this? Or do we want to commit the changes only when saving and existing edit mode?